src/main/resources/assets/io/github/nucleuspowered/nucleus/messages.properties

```diff
@@ -248,6 +248,10 @@ command.god.error=&cCould not set invulnerability status.
 command.hat.success=&aSet hat of {0} to {1}.
 command.hat.error.handempty=&cYou must be holding an item to use this command.
 
+command.lightning.success=&aLightning strike created!
```

Comment:
Oh, yeah, we need to be careful with keys. When I merged #89 today, I had to alter several keys that are like this to make it work properly. Here, `command.lightning.success` should be something like `command.lightning.success.normal`, because when I have my autogenerating thing convert this into a messages file, you can't have `command.lightning.success` as a root node and a string value in Configurate.

This needs changing to add an extra part to the path.